src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/QueryHelper.cs

```diff
@@ -82,7 +78,7 @@ public int GetProvidersTotalAmount()
                 _elasticsearchCustomClient.Search<Provider>(
                     s =>
                         s.Index(_applicationSettings.ProviderIndexAlias)
-                            .Type(Types.Parse(_providerDocumentType))
+                            .Type(Types.Parse("providerapidocument"))
```

Comment:
can we have it as a const?

Reply:
Updated the other one but this is only used once.